site/_layouts/post.html

```diff
@@ -0,0 +1,80 @@
+<!-- Blog Layout Start -->
+{% assign abc = page.author | split: "," %}
+{% if abc.size > 1 %}
+{% assign img_link = abc.last %}
+{% else %}
+{% assign img_link = "http://pageanthive.com/images/empty-profile.png" %}
+{% endif %}
+{% assign name = abc.first %}
+<!DOCTYPE html>
+<html lang="{{ site.lang }}">
+
+  {% include head.html %}
+
+  <body id="page-top" data-spy="scroll" data-target=".navbar"  data-offset="151">
+
+    {% include navigation.html %}
+    {% if page.title == 'Blog' %}
+    <div class="container-fluid" id="topCont">
+      <div class="row">
+        <img src="{{ 'assets/images/blogTop.png' | relative_url }}">
+      </div>
+    </div>
+    {% endif %}
+
+    <div class="container" id="postsContainer">
+      <div class="row">
+    {% if page.title != 'Blog' %}
+      <div class="col-md-10 col-lg-10 col-md-offset-1 col-lg-offset-1">
+        <div class="row blogPost">
+          <div class="col-md-12 col-lg-12">
+            <div class="row postTop">
+              <p class="timeStamp">{{ name }} blogged on {{ page.created_at | date: "%a, %d %b %Y" }}</p>
+              <img class="postWriter" src="{{ img_link }}" style="min-width:150px; min-height:150px; max-height:150px; max-width:150px;">
+              <p class="postTitle">{{ page.title }}</p>
```

Comment:
Somewhere alongside the post title, a link to the original blog post?

Reply:
This could be a separate issue

Reply:
I too want this to be done, but it is limited by the scope of the default gem
I have the changes done with regards to this gem on my fork, and ideally would've wanted this to be integrated upstream, but they replied to me on their discussion forum that we might want to customize the gem to our needs and use it separately

I'll have to look into this a little bit later, meanwhile maybe you might get the technical insight from here: https://github.com/virresh/jekyll-planet

Reply:
Forum discussion link?

Reply:
https://groups.google.com/forum/#!topic/wwwmake/af7aVT5igfI